tests/migrate/fix_course_coach.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/* For licensing terms, see /license.txt */
+/**
+ * Set course coach if there is not assigned already one 
+ * @author Angel Fernando Quiroz Campos <angel.quiroz@beeznest.com>
+ * @package chamilo.migration
+ */
+if (PHP_SAPI != 'cli') {
+    exit;
+}
+
+require_once dirname(__FILE__) . '/../../main/inc/global.inc.php';
+
+$year = '2015';
+
+$sessionTable = Database::get_main_table(TABLE_MAIN_SESSION);
+$sessionCourseTable = Database::get_main_table(TABLE_MAIN_SESSION_COURSE);
+$sessionCourseUserTable = Database::get_main_table(TABLE_MAIN_SESSION_COURSE_USER);
+
+$sql = "SELECT s.id, s.id_coach, sc.course_code 
+    FROM $sessionTable s 
+    INNER JOIN $sessionCourseTable sc ON s.id = sc.id_session 
+    WHERE YEAR( s.display_start_date ) >= $year 
+    AND s.id NOT IN( 
+        SELECT scu.id_session 
+        FROM $sessionCourseUserTable scu 
+        INNER JOIN $sessionTable s2 ON scu.id_session = s2.id 
+        WHERE scu.status = 2 AND 
+        YEAR( s2.display_start_date ) >= $year
+    )";
```

Comment:
The query is very specific to this user as it will only work for sessions that only include one single course. Make sure you put a comment that mentions that (if anyone would like to extend on this).
Also comment on scu.status = 2: what does it mean (to improve readability of your code). I suppose it means there's a course coach user?